src/syrupy/extensions/amber/serializer.py

```diff
@@ -337,6 +341,14 @@ def serialize_dict(
             **kwargs,
         )
 
+    @classmethod
+    def serialize_function(
+        cls, data: FunctionType, *, depth: int = 0, **kwargs: Any
+    ) -> str:
+        return cls.__serialize_plain(
+            data=f"{data.__qualname__}{str(inspect.signature(data))}", depth=depth
```

Comment:
Looking at the doc for inspect.signature again, I'm wondering if I should set `eval_str=False`. @noahnu , thoughts?

https://docs.python.org/3/library/inspect.html#inspect.signature

Reply:
It defaults to false, so no action required.

Reply:
Oh, right. lol.